### DISCUSSION TITLE
Strange google maps theming on emulator

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#15104] [Q&A]
Since clean up of my emulators (due to the package renaming) I have the problem, that Google maps (on unfied map) shows normal on lower zooms but seemingly starts using high contrast on higher zoom.

Any idea why? Could not reproduce this bug on a real device.

Low zoom:
![image](https://github.com/cgeo/cgeo/assets/949669/fbdc5826-4616-4127-b465-569ac64edb54)

Higher zoom:
![image](https://github.com/cgeo/cgeo/assets/949669/bf6658ff-758e-4df3-ac58-dd913c2a7855)

**Comment by moving-bits** [comment]
Haven't seen this yet. Maybe some sort of caching problem with temporary data from Google Maps? Can this cache be deleted? (where does the integrated GM store such temporary data? is it app-specific? - then deleting c:geo app cache may help - or global?)

![image](https://github.com/cgeo/cgeo/assets/3754370/e1068ff6-16e9-44ba-b760-348270fcb73a)![image](https://github.com/cgeo/cgeo/assets/3754370/709ad8ff-5175-48ff-8a50-702aabd27ac4)![image](https://github.com/cgeo/cgeo/assets/3754370/2823b00e-f3e3-491d-bced-3d13aa232ee4)

**Comment by Lineflyer** [comment]
I never figured out completely what was going on here, but:
The problem vanished after disabling IPv6 connection to the internet. This probably means, that at that time either the Google servers or the local emulator engine had some kind of problem to send/receive proper map data when communication via IPv6.

Just documenting, what I found out at that time in case someone runs into a similar problem in future.
I did not check whether the problem still appears on IPv6 though.
